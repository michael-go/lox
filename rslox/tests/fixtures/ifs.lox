if (true) {
    print "yes";
} else {
    print "no";
}

if (false) {
    print "bad";
} else {
    print "good";
}