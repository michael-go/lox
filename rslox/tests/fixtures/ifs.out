# exit code: 0
# stdout:
"yes"
"good"

# stderr:

